test/Microsoft.NET.Sdk.Functions.Generator.V2.Tests/Microsoft.NET.Sdk.Functions.Generator.V2.Tests.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Since this is in the context of Functions (rather than webjobs) -- should we call this project V1?

Reply:
Changed it to Microsoft.NET.Sdk.Functions.Generator.V1.Tests